lib/compass-rails.rb

```diff
@@ -35,14 +35,14 @@ def sass_config
     end
 
     def sprockets
-      @sprockets ||= ::Rails.application.assets
+      @sprockets ||= ::Rails.application.assets || ::Rails.application.config.assets
```

Comment:
You are right, however, they both respond to paths. Nevermind, I think @timhaines took a better approach at fixing this issue in #263.